sql/org_mozilla_firefox/baseline_daily/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_daily`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.org_mozilla_firefox_derived.baseline_daily_v1`
```

Comment:
The ETL here is across all channels of Fenix; I chose to put them in `org_mozilla_firefox` as the production app name, even though they hit datasets for a variety of distinct app names for the different channels. @fbertsch expressed some concern about this, but it seems like a reasonable compromise to avoid the extra overhead of defining these queries separately for each of the 5 app names involved and scheduling all of the queries.